CMakeLists.txt

```diff
@@ -83,11 +83,21 @@ if(NOT CMAKE_CONFIGURATION_TYPES)
     set_property(CACHE CMAKE_BUILD_TYPE PROPERTY STRINGS ${ROBOTOLOGY_BUILD_TYPES})
 endif()
 
-# Optionally load a file named ProjectsTagsMaster
-# for overwriting the desired branch/tag/commit
-# of the included subprojects of the superbuild
-# Useful for having superbuild "branches"
-include(ProjectsTagsMaster OPTIONAL)
+set(ROBOTOLOGY_PROJECT_TAGS "Stable" CACHE STRING "The tags to be used for the robotology projects: Stable, Unstable or Custom. This can be changed only before the first configuration.")
+mark_as_advanced(ROBOTOLOGY_PROJECT_TAGS)
+set(ROBOTOLOGY_PROJECT_TAGS_CUSTOM_FILE CACHE FILEPATH "If ROBOTOLOGY_PROJECT_TAGS is custom, this file will be loaded to specify the tags of the projects to use.")
+mark_as_advanced(ROBOTOLOGY_PROJECT_TAGS_CUSTOM_FILE)
+set_property(CACHE CMAKE_BUILD_TYPE PROPERTY STRINGS "Stable" "Unstable" "Custom")
```

Comment:
shouldn't it be `ROBOTOLOGY_PROJECT_TAGS` instead od `CMAKE_BUILD_TYPE` here?

Reply:
Good catch! This was a copy paste error.